resources/Makefile.erb

```diff
@@ -1,11 +1,18 @@
 <% dirnames = get_directories.map {|d| d.path } %>
-export SHELL := $(shell which bash)
 
-tempdir := $(shell mktemp -d -p /var/tmp 2>/dev/null || mktemp -d -t 'tmp')
 workdir := $(PWD)
 
+export SHELL := "${workdir}/profiling_shell.sh"
+export VANAGON_STATSD_HOST ?= 10.32.162.156
+export VANAGON_STATSD_PORT ?= 8125
+export VANAGON_PROJECT ?= <%= @name %>
+export VANAGON_PLATFORM ?= <%= @platform.name.tr('-','.') %>
```

Comment:
This means we'll be able to break out build stats for platform type, version, and architecture. It might be more detail than we need, but it might be nice to have the option to drill down to this if we want to.